packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -268,9 +300,8 @@ export const makeInnerVault = (
       locked: getCollateralAmount(),
       debt: getDebtAmount(),
       collateralizationRatio,
-      // TODO state distinct from CLOSED https://github.com/Agoric/agoric-sdk/issues/4539
-      liquidated: vaultState === VaultState.CLOSED,
-      vaultState: vstate,
+      liquidated: phase === VaultPhase.LIQUIDATED,
```

Comment:
I suspect the UI actually wants to know "is the vault closed or liquidated?".  "closed" is a better term for that than "liquidated", but maybe the UI needs to distinguish `[ [Active], [Closed, Liquidated], [Transferring]]`.  I think `Liquidating` is most like `Closed` and `Liquidating` among those, but we should probably check with @samsiegart.

Reply:
I'm assuming the end user would want to know all of these states. I'd really like to remove this flag entirely since it's redundant with the richer option, but I was trying to maintain backwards compatibility for now. To that end, I suppose `CLOSED` achieves that more!

@samsiegart is this backwards compatibility necessary or can we drop the `liquidated` flag in this PR? If not now then when would be good?

Reply:
from convo today, decision was to drop this boolean and rely just on the phase/state value